src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterGood.java

```diff
@@ -29,7 +29,7 @@ public void check3(int... a) {
 
     public void check4() throws java.io.IOException {
         try (java.io.InputStream ignored = System.in) {}
-        try {} catch(Exception e){}
+        try {} catch (Exception e){}
```

Comment:
This change should be in main commit `Issue #..`, since it is related to your changes.

Reply:
Yes, this commit should be only for indentation

Reply:
Do I have to make changes? This indentation fix was not related to my issue.

Reply:
Do change, but do this change in another commit

Reply:
> Do I have to make changes? This indentation fix was not related to my issue.

This became a violation when you added `LITERAL_CATCH` token to this check, so it is part of this issue :)

Reply:
Done

Reply:
Resolved.